roles/confluent.common/tasks/redhat.yml

```diff
@@ -40,13 +40,10 @@
   command: yum clean all
   args:
     warn: false
-  when: confluent_dist_repo_result.changed|default(False) or confluent_repo_result.changed|default(False) or custom_repo_result.changed|default(False) or repository_configuration == 'custom'
-
-- name: yum-makecache
-  command: yum makecache
-  args:
-    warn: false
-  when: confluent_dist_repo_result.changed|default(False) or confluent_repo_result.changed|default(False) or custom_repo_result.changed|default(False) or repository_configuration == 'custom'
+  when: >
+    confluent_dist_repo_result.changed|default(False) or
+    confluent_repo_result.changed|default(False) or
+    repository_configuration == 'custom'
```

Comment:
@domenicbove Note that the play will always run `yum clean all` if the user has a custom repository configuration. I am hoping this still will mend #785.

Reply:
Hi @erikgb , 
I'll try to test the PR without using the cache and let you know, you probably don't need to build a cache for yum but I don't understand the phrase "yum makecache tasks - which seems to slow down our pipeline significantly and requires a lot more disk space for the yum cache than it used to ", yum makecache download and make usable all the metadata for the currently enabled yum repositories. 
When you run `yum install/update <packege>` you still have to consider the times for downloading the metadata from all the enabled repo, what `yum makecache` would do , so in terms of space, but above all of time, not much changes.

I keep you updated.

Reply:
@22RC Thanks for trying this out! I can promise you that this makes a huge difference in our environment. It seems like `yum makecache` does a lot more than just installing a package starting with an empty cache. I found these references to this issue:

- https://forums.centos.org/viewtopic.php?t=77580
- https://access.redhat.com/solutions/5302591

Please let me know how this works out for you!

IMO the long term solution should be to avoid the non-idempotent `yum clean all`, and instead add `update_cache: true` to all yum tasks. Ref. https://docs.ansible.com/ansible/latest/collections/ansible/builtin/yum_module.html#parameter-update_cache. But that will be a follow-up PR once Confluent has resolved their block-community issue.

We tried to upgrade to CP 7.0.0 - and then `yum makecache` gets even worse - as there is a new RPM repo introduced for client packages. 💣